source/documentation/deploying_services/index.md

```diff
@@ -1,19 +1,23 @@
 # Deploy a backing service
 
-## Backing services overview
+## Services and plans
 
-
-Your application probably relies on backing services such as a database, an email delivery service or a monitoring system.
+Many 12-factor applications rely on backing services such as a database, an email delivery service or a monitoring system.
 
 In Cloud Foundry, backing services are referred to as 'services' and are available through the Cloud Foundry ``cf marketplace`` command.
 
 Currently, the only service available is the [PostgreSQL database service](/#postgresql). 
 
-### Paid services
+Each service can have multiple plans available. For example, there are different PostgreSQL plans which vary by availability, storage capacity and encryption.
```

Comment:
as of now, all our postgres plans have same, 20GB storage capacity. If we mention it varies, people might ask about it. I'd simply change this to 'capacity' instead of 'storage capacity'. That would then also mean computational/throughput capacity - or any other service specific capacity in general.